code/modules/mob/living/carbon/xenomorph/embryo.dm

```diff
@@ -77,10 +77,10 @@
 	if(CHECK_BITFIELD(affected_mob.restrained_flags, RESTRAINED_XENO_NEST)) //Hosts who are nested in resin nests provide an ideal setting, larva grows faster.
 		counter += 1 + max(0, (0.03 * affected_mob.health)) //Up to +300% faster, depending on the health of the host.
 	else if(stage <= 4)
-		counter++
+		counter += 2 //Doubled growth overall, lowering free burst time to ~10 min.
```

Comment:
This comment is referring to your current change. Future coders checking this won't really care there was a change.
Keep it informative to the current situation. Something like this:
```suggestion
		counter += 2 //Free burst time in ~10 min.
```